include/deal.II/base/patterns.h

```diff
@@ -1500,11 +1500,11 @@ namespace Patterns
           return std_cxx14::make_unique<Patterns::Bool>();
         else if (std::is_integral<T>::value)
           return std_cxx14::make_unique<Patterns::Integer>(
-            static_cast<int>(std::numeric_limits<T>::min()),
+            static_cast<int>(std::numeric_limits<T>::lowest()),
```

Comment:
For a moment I thought this was a bug because `min()` and `lowest()` return different values for floating point numbers. But for integral types, they seem to yield the same: https://en.cppreference.com/w/cpp/types/numeric_limits